### ISSUE TITLE
Dynamic imports support

### ISSUE THREAD

**Issue by manuelmazzuola** [shimarulin/vuettext#14]
When the script tries to parse this line `{ component: () => import(/* webpackChunkName: "orders" */ './views/Orders.vue'), }` it fails with
```
{ SyntaxError: Unexpected token (29:23)
    at Parser.pp$4.raise (/home/originof/src/backend-ui/node_modules/vuettext/node_modules/acorn/dist/acorn.js:2845:13)                                      
    at Parser.pp.unexpected (/home/originof/src/backend-ui/node_modules/vuettext/node_modules/acorn/dist/acorn.js:690:8)                                     
    at Parser.pp$3.parseExprAtom (/home/originof/src/backend-ui/node_modules/vuettext/node_modules/acorn/dist/acorn.js:2292:10)                              
    at Parser.pp$3.parseExprSubscripts (/home/originof/src/backend-ui/node_modules/vuettext/node_modules/acorn/dist/acorn.js:2126:19)                        
    at Parser.pp$3.parseMaybeUnary (/home/originof/src/backend-ui/node_modules/vuettext/node_modules/acorn/dist/acorn.js:2103:17)                            
    at Parser.pp$3.parseExprOps (/home/originof/src/backend-ui/node_modules/vuettext/node_modules/acorn/dist/acorn.js:2045:19)                               
    at Parser.pp$3.parseMaybeConditional (/home/originof/src/backend-ui/node_modules/vuettext/node_modules/acorn/dist/acorn.js:2028:19)                      
    at Parser.pp$3.parseMaybeAssign (/home/originof/src/backend-ui/node_modules/vuettext/node_modules/acorn/dist/acorn.js:2003:19)                           
    at Parser.pp$3.parseFunctionBody (/home/originof/src/backend-ui/node_modules/vuettext/node_modules/acorn/dist/acorn.js:2664:22)                          
    at Parser.pp$3.parseArrowExpression (/home/originof/src/backend-ui/node_modules/vuettext/node_modules/acorn/dist/acorn.js:2649:8)                        
  pos: 704,
  loc: Position { line: 29, column: 23 },
  raisedAt: 710 }
```

**Comment by manuelmazzuola** [comment]
acorn issue https://github.com/acornjs/acorn/issues/823

**Comment by manuelmazzuola** [comment]
This package should use https://www.npmjs.com/package/acorn-dynamic-import to support dynamic imports.

**Comment by shimarulin** [comment]
Thank you @manuelmazzuola for your contributing! You pull requests #17 and #19 also merged to master and released in version 0.6.0
